site/docs/skylark/skylint.md

```diff
@@ -139,6 +139,27 @@ def example_function2(foo, bar):
     This function is deprecated for <reason>. Use <alternative> instead.
   """
   return "baz"
+
+def _new_rule_impl(ctx):
+  print("new_rule")
+
+new_rule = rule(
+  _new_rule_impl
+  attrs = {
+    "foo": attr.label(
+      doc = """documentation of the first attribute.
+
+with multiline description
+"""),
+    "bar": attr.string(
+      doc = "documentation of the second attribute"
+    ),
+  },
+  doc = """Illustrates the format for rule definitions
+
+After the one-line summary comes the description body.
+It contains additional information about the rule.
+""")
 ```
 
 #### Indentation
```

Comment:
There should also be a new subsection of the docstrings section describing how to document rules and rule attrs. (Unlike for functions, there's no precise format for the string content, but we need to at least say that the `doc` param should be present.) This new subsection can be the anchor target of the new issue types `missing-rule-documentation` and `missing-attribute-documentation` (see use of `<a>` tags for the existing sections).